cluster/engine.go

```diff
@@ -470,8 +470,14 @@ func (e *Engine) updateClientVersionFromServer(serverVersion string) {
 		e.apiClient.UpdateClientVersion("1.23")
 	case versions.LessThan(serverVersion, "1.13"):
 		e.apiClient.UpdateClientVersion("1.24")
-	default:
+	case versions.LessThan(serverVersion, "1.13.1"):
 		e.apiClient.UpdateClientVersion("1.25")
+	case versions.LessThan(serverVersion, "17.03.1"):
```

Comment:
@allencloud good point. I wonder if this is something we need to start checking for everywhere (stripping off the ~ce) at the end if it's going to be present.

WDYT @dongluochen?